cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -0,0 +1,56 @@
+Crontasks settings
+==================
+
+You find here, some good practices about crontasks management in Akeneo PIM Flexibility Edition :
```

Comment:
You'll find here
Edition:

Reply:
This is not really good practices here you are just describing how it works ;)

Reply:
I'd even go with  "Here you will find some good practices ..."

Reply:
more simple :
You'll find here some infos about crontasks management in Akeneo PIM Flexibility Edition:

Reply:
no "s" to info :) maybe you should use "information"